### DISCUSSION TITLE
Balance displays with invalid decimals

### DISCUSSION THREAD

**Discussion by cclub1** [Steffo99/greed#149] [Q&A] [answered]
hello steffo,

Great work with greed!

But i have a problem. The wallet shows balance 1.00 in stead of 100.00. Do you know how I can solve this?

**Comment by Steffo99** [comment, marked as answer]
Sounds like a problem with the `currency_exp`. What is your current config?

**Reply by cclub1** [reply]
My current setting was on 2. i changed it to 0. Problem solved! Thank you for your fast response!
